app/poros/save_request/skip_checker.rb

```diff
@@ -0,0 +1,21 @@
+class SaveRequest::SkipChecker
```

Comment:
I am feeling super happy with my decision to create `app/poros/`, where I can just throw things like this. In that sense, it's sort of a junk drawer, but I don't mind that. Many people might just call it `app/services/`, but I sort of like the low pressure and informality of "POROs". It makes me feel free to put in here whatever I think will be most useful.